node-operators/networks/compile-binary.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Compile the Binary
+description: How to compile the Moonbeam binary to run a full Parachain node, gain access to RPC endpoints, and produce blocks, for the Moonbeam Network.
+---
+
+# Compile the Moonbeam Binary
+
+![Full Node Moonbeam Banner](/images/fullnode/compile-binary-banner.png)
+
+## Introduction
+
+There are several ways to get started running a full node on the Moonbeam network. This guide goes through the process of compiling the Moonbeam binary to be run as a systemd service. For a more general overview of running nodes, or to get started with Docker, check out the [Run a Node](/node-operators/networks/full-node) page of our documentation.
+
+This guide is meant for people with experience running [Substrate](https://substrate.dev/) based chains. Running a parachain is similar to running a Substrate node with a few differences. A Substrate parachain node will run two processes: one to sync the relay chain and one to sync the parachain. As such, many things are doubled, for example, the database directory, the ports used, the log lines, and more.
```

Comment:
This seems more relevant for the running a node page than the compiling page.

Reply:
Although the Substrate part is definitely relevant...

```suggestion
This guide is meant for people with experience compiling [Substrate](https://substrate.dev/) based blockchain nodes. A parachain node is similar to a typical Substrate node, but there are some differences. A Substrate parachain node will is a bigger build because it contains code to run the parachain itself, as well as code to sync the relay chain, and facilitate communication between the two. As such, this build is quite large and may take over 30 min and 32GB memory.
```